polaris-core/src/main/java/org/apache/polaris/core/config/FeatureConfiguration.java

```diff
@@ -131,7 +131,7 @@ public static void enforceFeatureEnabledOrThrow(
       PolarisConfiguration.<Boolean>builder()
           .key("ALLOW_EXTERNAL_TABLE_LOCATION")
           .catalogConfig("polaris.config.allow.external.table.location")
-          .catalogConfigUnsafe("allow.external.table.location")
+          .legacyCatalogConfig("allow.external.table.location")
```

Comment:
I still like `unsafe` because it suggests the unsafe nature of the method -- it's not clear to me what "legacy" means here. It's old, but, what's different about it?